contrib/tmpfile/podman.conf

```diff
@@ -6,3 +6,6 @@ x /tmp/run-*/libpod
 D! /var/lib/containers/storage/tmp 0700 root root
 D! /run/podman 0700 root root
 D! /var/lib/cni/networks
+# Remove /var/tmp/oci* and /var/tmp/storage* podman temporary directories on each
+# boot which are created when creating containers from oci-archive tarballs
+R! /var/tmp/container_images*
```

Comment:
Why only at boot? I feel like if you have a long running server we can clean these up much earlier. I would even say set this to 1 day.

Reply:
Well then you would have a risk of someone pulling an image and then systemd kills it.
These should only leak when something is killed, so it should not be that common.

Reply:
No that is not what tmpfiles does. It will only removes files that are not touched in the last X time.
> The date field, when set, is used to decide what files to delete when cleaning. If a file or directory is older than the current time minus the age field, it is deleted.

So I don't see how systemd would ever delete files that are still in use by us.

Reply:
I don't see how you can use a timer with a glob to remove files?

Reply:
Yeah not 100% sure either, I don't have time to look into it but this change works as well so fine for me.